execution/executor/tests/storage_integration_test.rs

```diff
@@ -33,7 +33,8 @@ use std::convert::TryFrom;
 use storage_interface::DbReaderWriter;
 use transaction_builder::{
     encode_block_prologue_script, encode_mint_script, encode_publishing_option_script,
-    encode_rotate_consensus_pubkey_script, encode_transfer_with_metadata_script,
+    encode_reconfigure_script, encode_set_validator_config_script,
```

Comment:
as a big to do, we need to move these out of storage_integration_tests and create a proper LibraMoveE2E testsuite in libra/testsuite/libra_move_interface or something